platform/lib/routers/pages.js

```diff
@@ -93,7 +93,7 @@ if (config.isDevMode()) {
    * @return {Boolean}
    */
   async function hasManualFormatVariant(request, format) {
-    const path = request.originalUrl.replace('.html', `.${format}.html`);
+    const path = request.url.replace('.html', `.${format}.html`);
```

Comment:
I'm remember there was a reason why I changed it to originalUrl

Reply:
Did you test this with `run start:local` and `run develop` ?

Reply:
Now tested both, but the change would actually only have an effect for `if (config.isDevMode())` - checking for a manual variant does work a little bit differently for other environments 🙂

Reply:
didn't notice, fine then